packages/blade/src/utils/makeAccessible/webToNativeRole.ts

```diff
@@ -0,0 +1,27 @@
+const webToAccessibilityRole = {
```

Comment:
what is the intent of this var? finding it hard to understand 🤔

Reply:
so this var is basically the web a11y roles? if yes, then we can probably rename it to `webAccessibilityRoles`?

Reply:
No not web. It's `web -> native`

[This](https://github.com/razorpay/blade/pull/481#discussion_r900022908)

For example: 
Let's say you are building a <Slider \/> component and you have to put proper aria role for it. 

In web we have the [`slider`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/slider_role) role
but in native it's called [`adjustable`](https://reactnative.dev/docs/accessibility#accessibilityrole)

So we are unifying out makeAccessible() API to be consistent. 

if we say:
```
makeAccessible({ role: 'slider' }) 
```
This will return `slider` for web and `adjustable` for native.

Reply:
let's discuss this in person